src/preferences.html

```diff
@@ -124,7 +124,11 @@
         return;
       }
       const syncInterval = parseInt(githubSyncInterval.value, 10);
-      ipcRenderer.send('setSyncInterval', syncInterval);
+      if (syncInterval > 0) {
```

Comment:
I meant adding a `min` value on the html `input` element, then we don't need to do backend validation here 😄

Reply:
Added, but it still need this validation because we can still enter "0" in it, and then Streaker will crash because of too many requests sent

Reply:
True!